vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/database/database_constants.h

```diff
@@ -10,8 +10,8 @@
 
 namespace ads::database {
 
-constexpr int32_t kVersion = 24;
-constexpr int32_t kCompatibleVersion = 24;
+constexpr int32_t kVersion = 25;
```

Comment:
Database tables in database_schema_25.sqlite should not have any entries. Thanks